decryptor/postgresql/prepared_statements_sql_observer.go

```diff
@@ -120,12 +131,21 @@ func (encryptor *PreparedStatementsQuery) onExecute(ctx context.Context, execute
 	}
 
 	// collect all BoundValues
-	values := make([]base.BoundValue, 0, len(executeQuery.Values))
-	for _, value := range executeQuery.Values {
-		switch val := value.(type) {
-		case *sqlparser.SQLVal:
-			values = append(values, NewPgBoundValue(val.Val, bindFormatText))
-		case *sqlparser.NullVal:
+	values := make([]base.BoundValue, 0, len(executeQuery.GetParams()))
+	for _, param := range executeQuery.GetParams() {
+		if param.GetAConst() == nil {
+			continue
+		}
+
+		switch {
+		case param.GetAConst().GetSval() != nil:
```

Comment:
please, add comments what means A/F/I values. From GetSVal I can guess that it's string value, but I'm not sure that correctly understand GetFval. Does it float value or function value...